stylesheets/helpers/utils.css

```diff
@@ -87,10 +103,43 @@
 .shift--small { margin-left: var(--control-height--small); }
 .shift--large { margin-left: var(--control-height--large); }
 
-/* BLOCKS */
-
 .stretch {
     width: 100%;
     justify-items: stretch;
 }
 
+/* POSITIONING */
+
+.sticky {
+    position: sticky;
+    z-index: 2;
```

Comment:
Imo z-scale should be either standardized or left outside the core (no objections to keep it as it is though)

Reply:
agree, I'll remove this one